### ISSUE TITLE
No `Latency` in `heartbeat()` callback

### ISSUE THREAD

**Issue by amingilani** [blinktrade/BlinkTradeJS#5]
The `.Latency` in the heartbeat callback is `undefined` unlike the `.Latency` in the promise

```js
            blinktrade.heartbeat()
                .then(function(beat) {
                    console.log(beat.Latency); //=> this returns the latency in ms
                });

            blinktrade.heartbeat(function(err, beat) {
                    console.log(beat.Latency); //=> this returns `undefined`
                });

```

**Comment by cesardeazevedo** [comment]
My bad, i'm currently exposing a better format on some return calls, and i think these formats are not working with callbacks, thanks for report, i'm gonna fix that...

**Comment by amingilani** [comment]
It's cool! You can't be expected to know about every bug! That's we you
have Github issues :)

**Comment by cesardeazevedo** [comment]
Yes indeed, i had to support callbacks to provide a flexible and idiomatic API, but it's lacking a better test coverage.

This fix has been shipped at 0.0.3, Keep me update if you have any other issues.
